### DISCUSSION TITLE
Just an appreciation post

### DISCUSSION THREAD

**Discussion by Alhizzohiz** [darkreader/darkreader#13078] [General]
Thanks Alex for making the best most god damn useful extension on the net. I make donations when I can but the fact it's still free to use after all this time speaks volumes of your character (I'm not attempting to sway you to not monetise this if you decide you need to). Made an account literally just to say this.

**Comment by 2zqa** [comment]
Saw the recent blogpost "How to fix the recent Google Search issue" and really made me feel bad for the main contributors and support handlers for all the negativity that washed over them. If you haven't read it yet, it's at the bottom of the Dark Reader extension pop-up, like all new blog posts.

Made my first donation as well. Lets bring that percentage of non-donating people down from 99.9% :muscle:
